packages/superset-ui-legacy-preset-chart-deckgl/src/DeckGLContainer.jsx

```diff
@@ -88,31 +91,38 @@ export default class DeckGLContainer extends React.Component {
     return this.props.layers;
   }
 
+  setTooltip = tooltip => {
+    this.setState({ tooltip });
+  };
+
   render() {
     const { children, bottomMargin, height, width } = this.props;
-    const { viewState } = this.state;
+    const { viewState, tooltip } = this.state;
     const adjustedHeight = height - bottomMargin;
 
     const layers = this.layers();
 
     return (
-      <div style={{ position: 'relative', width, height: adjustedHeight }}>
-        <DeckGL
-          initWebGLParameters
-          controller
-          width={width}
-          height={adjustedHeight}
-          layers={layers}
-          viewState={viewState}
-          onViewStateChange={this.onViewStateChange}
-        >
-          <StaticMap
-            mapStyle={this.props.mapStyle}
-            mapboxApiAccessToken={this.props.mapboxApiAccessToken}
-          />
-        </DeckGL>
-        {children}
-      </div>
+      <>
+        <Tooltip tooltip={tooltip} />
```

Comment:
should this be rendered on top of the div below? (not sure what z-index is like etc.)

Reply:
Good call. Will try moving it down.